components/alert/alert-toast.js

```diff
@@ -58,6 +61,9 @@ class AlertToast extends LitElement {
 			 * @default "default"
 			 */
 			type: { type: String, reflect: true },
+			_bottomHeight: { type: Number },
+			_bottomMargin: { type: Number },
```

Comment:
`_bottomHeight` and `_bottomMargin` feel like they could be state properties as well?